src/components/inputs/content-switcher/content-switcher.component.tsx

```diff
@@ -78,6 +83,17 @@ const ContentSwitcher: React.FC<FormFieldProps> = ({ question, onChange, handler
             />
           ))}
         </CdsContentSwitcher>
+
+        {question.questionOptions.showDate === 'true' ? (
+          <div style={{ marginTop: '5px' }}>
+            <InlineDate
+              question={question}
+              setObsDateTime={(value) => setObsDate(value)}
+            />
+          </div>
+        ) : (
+          ''
```

Comment:
```suggestion
          null
```

Reply:
We should prefer using `null`, I believe `''` would still create a DOM node containing an empty string